src/packages/generator-helper/src/dmmf.ts

```diff
@@ -162,7 +163,7 @@ export namespace DMMF {
   export interface SchemaFieldDeprecation {
```

Comment:
Maybe we could rename this to `SchemaDeprecation` as it made an entry into `SchemaArg`

Reply:
I have changed it to `Deprecation` as it is applicable to both the `SchemaArg` and the `SchemaField`